File: content/examples/other/leetcode/problems/valid-anagram/example-1/solution.js
function isAnagramSort(s, t) {
  if (s.length !== t.length) return false;
  const a = s.split("").sort().join("");
  const b = t.split("").sort().join("");
  return a === b;
}

if (require.main === module) {
  console.log(isAnagramSort("anagram", "nagaram"));
  console.log(isAnagramSort("rat", "car"));
}

module.exports = { isAnagramSort };

File: content/examples/other/leetcode/problems/valid-anagram/example-1/solution.py
def is_anagram_sort(s, t):
    if len(s) != len(t):
        return False
    return sorted(s) == sorted(t)


if __name__ == "__main__":
    print(is_anagram_sort("anagram", "nagaram"))
    print(is_anagram_sort("rat", "car"))
